discord_lib/discord_chat_service.py

```diff
@@ -69,3 +69,7 @@ def get_or_create_channel(self, name, chan_type=ChannelType.GUILD_TEXT):
         else:
             channel = self.create_channel(name, chan_type)
         return channel
+
+    def create_invite_link(self, channel_id):
+        invite = self._client.channels_invites_create(channel_id, max_age=0)
+        return "https://discord.gg/{}".format(invite.code)
```

Comment:
You might want to use a Python f-string here, like: `f"https://discord.gg/{invite.code}"` since it's fewer characters and possibly easier to read.